spec/use_cases/provider_key_change_use_case_spec.rb

```diff
@@ -68,14 +68,36 @@ module Backend
         let(:service_ids_with_old_key) { [7001, 7002] }
         let(:use_case) { ProviderKeyChangeUseCase.new(old_key, new_key) }
 
+        def exercise_provider_service_memoizer(provider_key, service_id)
+          # Call the Service class methods just
+          # to force the memoization of them with
+          # specific values
+          Service.default_id(provider_key)
+          Service.authenticate_service_id(service_id, provider_key)
+          Service.load(provider_key)
+          Service.load_by_id(service_id)
+          Service.list(provider_key)
+          Service.provider_key_for(service_id)
+        end
+
+        def build_provider_service_memoizer_keys(provider_key, service_id)
+          memoizer_keys = Memoizer.build_keys_for_class(Service,
```

Comment:
no need to define `memoizer_keys`. It's not used anywhere.

Reply:
That's true. I'll remove it